<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-- назад</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>-1</y>
      <width>461</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="server_settings">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background: #49b5b5;
	font: 14pt &quot;Franklin Gothic Heavy&quot;;
	border-width: 3px;
}

QPushButton:hover{
	background: #54d0d0;
	font: 16pt &quot;Franklin Gothic Heavy&quot;;
	border-width: 5px;
}</string>
       </property>
       <property name="text">
        <string>⚙️НАСТРОЙКИ СЕРВЕРА⚙️</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>
PushButton
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>
PushButton
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>20</y>
      <width>311</width>
      <height>581</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="ScrollingChannels">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>579</height>
      </rect>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>281</width>
        <height>681</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutWithChannels"/>
     </widget>
     <widget class="QScrollBar" name="ScrollChannelsBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>20</width>
        <height>581</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
